ui/framework/src/ui-framework/widgets/SimpleNavigationWidget.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Widget

```

Comment:
pretty much same coment i had for the SimpleToolWidget, maybe this should be ViewNavigationToolbarComposer?